### ISSUE TITLE
Tree is not rendered when is not in active tab

### ISSUE THREAD

**Issue by jgolebiewski** [500tech/angular-tree-component#239]
Angular version: 4.0.1
TreeComponent version: 3.2.4

If tree is not on the first tab (angular material) then is not rendered - tree-node-collection element is empty - otherwise  works fine.

**Comment by Rakesh-Harsha** [comment]
Facing the same issue with Angular4 Support changes.

**Comment by Gillardo** [comment]
I agree this is not working again

**Comment by LeahPike** [comment]
Here is an example demonstrating the issue:

[TreeExample.zip](https://github.com/500tech/angular-tree-component/files/905181/TreeExample.zip)

**Comment by michaelkleinhenz** [comment]
Similar issue here. With Angular 4.0.1 and 3.2.4, the tree is not rendering. With 3.2.3, it is working fine. I have this behaviour on an active tab/view as well.

**Comment by Gillardo** [comment]
@500tech-user any news on this??

**Comment by dhilgarth** [comment]
For me this is happening when the tree is in a bootstrap modal dialog

**Comment by adamkleingit** [comment]
OK I did 2 things:
1) Solve this automatically for trees not using virtual scroll
2) For trees using virtual scroll - need to call sizeChanged() on tree component if it became visible or changed it's size (for example when switching to the tab, or displaying a modal).

For example if using with bootstrap tabs:

```
  <tab heading="Tab 2" (select)="onSelectTab()">

  onSelectTab() {
    setTimeout(() => {
      this.treeComponent.sizeChanged();
    });
  }
```

Will be released in coming version

**Comment by adamkleingit** [comment]
Fixed in 3.3.0

**Comment by adrian-mitangi** [comment]
Hi,
When will this be available on npm?  Right now, the latest is 3.3.1
